src/pyhf_combine_converter/pyhf_convert_to_datacard.py

```diff
@@ -10,7 +10,7 @@
 import json
 from optparse import OptionParser
 try:
-    from HiggsAnalysis.CombinedLimit.DatacardParser import *
+    import HiggsAnalysis.CombinedLimit.DatacardParser as DP
```

Comment:
```suggestion
```

@peterridolfi As you never actually use the `DP` module in `pyhf_convert_to_datacard.py`

```console
$ git grep "DP" -- src/
src/pyhf_combine_converter/pyhf_convert_to_datacard.py:    import HiggsAnalysis.CombinedLimit.DatacardParser as DP
src/pyhf_combine_converter/pyhf_converted_from_datacard.py:    import HiggsAnalysis.CombinedLimit.DatacardParser as DP
src/pyhf_combine_converter/pyhf_converted_from_datacard.py:DP.addDatacardParserOptions(parser)
src/pyhf_combine_converter/pyhf_converted_from_datacard.py:        DC = DP.parseCard(file=dc_file, options=options)
```

I think that you can remove this line unless there is something else that you were importing from the `HiggsAnalysis.CombinedLimit.DatacardParser` that this change misses.